Project-Aurora/Project-Aurora/Project-Aurora.csproj

```diff
@@ -932,6 +940,69 @@
     <Content Include="Profiles\Overlays\SkypeOverlay\Event_SkypeOverlay.cs" />
     <Content Include="Profiles\Overlays\SkypeOverlay\SkypeOverlaySettings.cs" />
     <Content Include="Profiles\Overlays\SkypeOverlay\State_SkypeOverlay.cs" />
+    <None Include="kb_layouts\Extra Features\keyboard101_numpad_right_features.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Include="kb_layouts\Extra Features\keyboard102_numpad_right_features.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Include="kb_layouts\Extra Features\keyboard21br_numpad_left_bottom_features.json">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
Why are some of these not PreserveNewest?

Reply:
Thank You for Your Suggestion. I am going to change this file.